extensions/postgres-flyway/src/main/resources/db/migration/V12__DROP_duplicate_indices.sql

```diff
@@ -0,0 +1,30 @@
+--
+--  Copyright (c) 2024 sovity GmbH
+--
+--  This program and the accompanying materials are made available under the
+--  terms of the Apache License, Version 2.0 which is available at
+--  https://www.apache.org/licenses/LICENSE-2.0
+--
+--  SPDX-License-Identifier: Apache-2.0
+--
+--  Contributors:
+--       sovity GmbH - Improve database performance by removing duplicate indices
+--
+
+-- Drop the non _pk index for edc_contract_agreement if it exists.
```

Comment:
nit pick. I would rather explain it as "Dropping duplicate pk index... " so I'm not afraid that this has a performance impact.

And they could be grouped in a single comment because it's the same thing for each line like "Drop the duplicated indexes if they exist" and maybe give a reason why (storage space?).